src/main/java/com/adyen/model/AccountContainer.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2017 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+package com.adyen.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * AccountContainer
+ */
+public class AccountContainer {
+    @SerializedName("Account")
+    private Account account = null;
+
+    /**
+     * account
+     *
+     * @return account
+     */
+    public Account getAccount() {
+        return account;
+    }
+
+    public void setAccount(Account account) {
+        this.account = account;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        AccountContainer accountContainer = (AccountContainer) o;
+        return Objects.equals(this.account, accountContainer.account);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(account);
+    }
+
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("class AccountContainer {\n");
+
+        sb.append("    account: ").append(toIndentedString(account)).append("\n");
+        sb.append("}");
+        return sb.toString();
+    }
+
+    private String toIndentedString(Object o) {
```

Comment:
Just randomly browsing here, but this method already exists in `Util.toIndentedString()`; it seems to be copied all over the model classes ;-)

Reply:
Hi @epochcoder,

Thanks for the comment. You are totally right. It is auto-generated code from swagger so that is why. Let's improve this separately from this PR because we need to update all our models. If you want you could create a PR to us with the fix if you need it right now otherwise we will fix it after finalizing this PR.

Regards,
Rik
Adyen